### PR TITLE
has done

### PR DESCRIPTION
**Pull request by mitArt**
it's work

### CODE REVIEW

File: modules/1_opencv/src/opencv_coins.cpp [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+    findContours(thresh, contoursVector, RETR_EXTERNAL, CHAIN_APPROX_NONE);
+
+    int coinsSum(0);
+    double R(0);
+    std::vector<double> area;
+    for (int i(0); i < contoursVector.size(); i++) {
+        area.push_back(contourArea(contoursVector[i]));
+        sort(area.begin(), area.end());
+    }
+    double minRadius2 = sqrt(area[10] / CV_PI);
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp]: Please try to avoid hardcoded numbers. What if we had less coins?

Comment (mitArt) [reply modules/1_opencv/src/opencv_coins.cpp]: I thought I can use the number of coins to solve

Before "return" I wrote the second solution option without using the information of amount.

Comment (dkurt) [reply modules/1_opencv/src/opencv_coins.cpp]: Please remove all commented code

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -1,13 +1,63 @@
...
+        double R = sqrt(contourArea(contour) / CV_PI);
+        //std::cout << R << std::endl;  //radiuses of circles
+        R < 10 ? coinsSum++ : coinsSum += 2;
+    }
+    
+    
+    return coinsSum;
 
     // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
     CV_Error(Error::StsNotImplemented, "countCoins");
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp:39]: Please remove unreachable lines

**Comment by dkurt** [comment]
Looks like there is some issue with Travis.

**Pull request closed without merging**